systemd/csp-billing-adapter.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=CSP usage reporting
-Wants=billing-data-service.service
-After=billing-data-service.service
+Wants=network.target billing-data-service.service
```

Comment:
This is not correct, we require the network and want the billing data service, i.e. this should be:
```
Requires=network.target
Wants=billing-data-service.service
```

The service cannot run if there is no network.

Reply:
Rebased with these changes